### PR TITLE
[开源创新大赛 赛题6] rm , from \* @param(.*?),

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR types

Others 
### PR changes

Others 
### Description

rm , from `\* @param(.*?),`

### CODE REVIEW

File: paddle/phi/api/include/tensor.h [hunk 1]
```diff
@@ -435,7 +435,7 @@ class PADDLE_API Tensor final {
    * data type template argument
    *
    * @tparam T
-   * @param target_place, the target place of which the tensor will copy to.
+   * @param target_place the target place of which the tensor will copy to.
```
Comment (jiahy0825) [inline paddle/phi/api/include/tensor.h]: `the target place` -> `The target place`

File: paddle/phi/api/include/tensor.h [hunk 2]
```diff
@@ -444,17 +444,17 @@ class PADDLE_API Tensor final {
...
    * @return Tensor
    */
   Tensor copy_to(const Place& place, bool blocking) const;
 
   /**
    * @brief Transfer the source Tensor to current Tensor.
    *
-   * @param src, the source Tensor to be copied.
-   * @param blocking, Should we copy this in sync way.
+   * @param src the source Tensor to be copied.
```
Comment (jiahy0825) [inline paddle/phi/api/include/tensor.h]: `the source Tensor` -> `The source Tensor`

**Review by jiahy0825 — CHANGES_REQUESTED** [review]
Fix two typo errors pls~

**Review by jiahy0825 — CHANGES_REQUESTED** [review]
Fix two typo errors pls~

File: paddle/phi/api/include/tensor.h (hunk 1 above)
Comment (Liyulingyue) [inline paddle/phi/api/include/tensor.h]:
```suggestion
   * @param target_place The target place of which the tensor will copy to.
```

File: paddle/phi/api/include/tensor.h (hunk 2 above)
Comment (Liyulingyue) [inline paddle/phi/api/include/tensor.h]:
```suggestion
   * @param src The source Tensor to be copied.
```

**Review by jiahy0825 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**